UHResidentInformationAPI/V1/Factories/ResponseFactory.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using UHResidentInformationAPI.V1.Boundary.Responses;
+using Address = UHResidentInformationAPI.V1.Domain.Address;
+using AddressResponse = UHResidentInformationAPI.V1.Boundary.Responses.Address;
+using ResidentInformation = UHResidentInformationAPI.V1.Domain.ResidentInformation;
+using ResidentInformationResponse = UHResidentInformationAPI.V1.Boundary.Responses.ResidentInformation;
+
+namespace UHResidentInformationAPI.V1.Factories
+{
+    public static class ResponseFactory
+    {
+        public static ResidentInformationResponse ToResponse(this ResidentInformation domain)
+        {
+            return new ResidentInformationResponse
+            {
+                HouseReference = domain.HouseReference,
+                PersonNumber = domain.PersonNumber,
+                FirstName = domain.FirstName,
+                LastName = domain.LastName,
+                DateOfBirth = domain.DateOfBirth,
+                NINumber = domain.NINumber,
+                PhoneNumber = domain.PhoneNumber.ToResponse(),
+                Email = domain.Email.ToResponse(),
+                Address = domain.Address.ToResponse()
+            };
+        }
+        public static List<ResidentInformationResponse> ToResponse(this IEnumerable<ResidentInformation> people)
+        {
+            return people.Select(p => p.ToResponse()).ToList();
+        }
+
+        private static List<Phone> ToResponse(this List<Domain.Phone> phoneNumbers)
+        {
+            return phoneNumbers.Select(number => new Phone
+            {
+                PhoneNumber = number.PhoneNumber,
+                PhoneType = number.Type,
+                LastModified = number.LastModified.ToString("O", CultureInfo.InvariantCulture)
+            }).ToList();
+        }
+
+        private static List<Email> ToResponse(this List<Domain.Email> emailAddresses)
+        {
+            return emailAddresses.Select(email => new Email()
+            {
+                EmailAddress = email.EmailAddress,
+                LastModified = email.LastModified.ToString("O", CultureInfo.InvariantCulture)
```

Comment:
Similarly, with the two Last Modified fields - will they always be populated, otherwise, it'll throw an exception

Reply:
That makes sense.
The columns for this (`modDate`) have a `NOT NULL` description in the schema.
It might be safe to assume that these won't be null in the live DB.